src/utils/timeSeriesResultsByOption.js

```diff
@@ -88,6 +93,7 @@ function getMonthlyResult (
   mortgageRate,
   investmentRate,
   inflationRate,
+  investmentPMT,
   lastResult
```

Comment:
We are now passing the actual `investmentPMT` that is being made, which isn't inflation adjusted